Userland/Libraries/LibWeb/WebDriver/TimeoutsConfiguration.cpp

```diff
@@ -74,7 +75,7 @@ ErrorOr<TimeoutsConfiguration, Error> json_deserialize_as_a_timeouts_configurati
     // 4. If value has a property with the key "pageLoad":
     if (value.as_object().has("pageLoad"sv)) {
         // 1. Let page load duration be the value of property "pageLoad".
-        auto page_load_duration = value.as_object().get_i64("pageLoad"sv);
+        auto page_load_duration = value.as_object().get_double_with_precision_loss("pageLoad"sv);
```

Comment:
LGTM but I think we should add a comment here saying why we're parsing as a double. Otherwise I'd get confused next time I look at this, because the spec says the page load duration is an integer.